packages/perseus/src/styles/widgets/expression.less

```diff
@@ -113,8 +113,6 @@
 }
 
 .perseus-widget-expression-old > .output > .placeholder  > .error {
-    -webkit-border-radius: 10px;
-    -moz-border-radius: 10px;
```

Comment:
Guessing these are no longer needed with evergreen browsers?

Reply:
Yeah, I don't think prefixes for border-radius have been needed for years. I'm tempted to file a PR where I prune all old vender-prefixes.

Reply:
I, for one, would heartily support that!!!

Reply:
Alrighty: https://github.com/Khan/perseus/pull/416